odfuzz/fuzzer.py

```diff
@@ -1115,20 +1115,35 @@ def add_predecessor(self, predecessor_id):
     def build_string(self):
     #TODO refactor rename build_url_part - this creates the parts after /Entity?$filter... etc ; not entire URL to send to Dispatcher.
         self._query_string = self._accessible_entity.path + '?'
-        for option_name in self._order:
-            if option_name.endswith('filter'):
-                filter_data = deepcopy(self._options[option_name[1:]])
-                option_string = build_filter_string(filter_data)
-            elif option_name.endswith('orderby'):
-                orderby_data = self._options[option_name[1:]]
-                orderby_option = OrderbyOption(orderby_data)
-                option_string = OrderbyOptionBuilder(orderby_option).build()
-            elif option_name.endswith('expand'):
-                option_string = ','.join(self._options[option_name[1:]])
-            else:
-                option_string = self._options[option_name[1:]]
-            self._options_strings[option_name[1:]] = option_string
-            self._query_string += option_name[1:] + '=' + option_string + '&'
+        # for option_name in self._order:
+        #     if option_name.endswith('filter'):
+        #         filter_data = deepcopy(self._options[option_name[1:]])
+        #         option_string = build_filter_string(filter_data)
+        #     elif option_name.endswith('orderby'):
+        #         orderby_data = self._options[option_name[1:]]
+        #         orderby_option = OrderbyOption(orderby_data)
+        #         option_string = OrderbyOptionBuilder(orderby_option).build()
+        #     elif option_name.endswith('expand'):
+        #         option_string = ','.join(self._options[option_name[1:]])
+        #     else:
+        #         option_string = self._options[option_name[1:]]
+        #     self._options_strings[option_name[1:]] = option_string
+        #     self._query_string += option_name[1:] + '=' + option_string + '&'
```

Comment:
This is really ugly. 

either uncomment - if this was leftover from some debugging session - or delete, if the code is not valid. I will read it thoroughly after such change.

Reply:
ok